ocfweb/lab_reservations/templates/lab_reservations/index.html

```diff
@@ -0,0 +1,84 @@
+{% extends "base.html" %}
+{% load bootstrap %}
+{% load staticfiles %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-8 ocf-content-block">
+
+            <form method="post" action="{% url 'request_reservation' %}">
+                {% csrf_token %}
+
+
+                <hr />
```

Comment:
Could you include a screenshot? It seems weird to me to have a horizontal line after nothing (the CSRF token is a hidden form field).